exercises/triangle/src/example.h

```diff
@@ -0,0 +1,13 @@
+#ifndef TRIANGLE_H
```

Comment:
Since we're defining enums here to use in the test cases, you need to rename the file to `triangle.h` so that this file is included in the download. See the Allergies exercise and #120. (Also, as a relatively new contributor, any input you may have on what we need to explain in #120 would be helpful).